nucypher/network/middleware.py

```diff
@@ -29,7 +30,7 @@ def __init__(
         self,
         eth_provider_uri: Optional[str],
         registry: Optional["BaseContractRegistry"] = None,
```

Comment:
Looks like this is imported already.

```suggestion
        registry: Optional[BaseContractRegistry] = None,
```

Reply:
Missed that one. Nice catch!